ttax/decompositions.py

```diff
@@ -0,0 +1,251 @@
+import numpy as np
+import jax
+import jax.numpy as jnp
+
+from ttax.base_class import TT
+from ttax.base_class import TTMatrix
+
+
+def round(tt, max_tt_rank=None, epsilon=None):
+  """TT-rounding procedure, returns a TT object with smaller TT-ranks.
+  Args:
+    tt: TT-tensor or TT-matrix.
+    max_tt_rank: a number or a list of numbers.
+      If a number, than defines the maximal TT-rank of the result.
+      If a list of numbers, than `max_tt_rank` length should be d+1
+      (where d is the number of dimensions) and `max_tt_rank[i]` defines
+      the maximal (i+1)-th TT-rank of the result.
+      The following two versions are equivalent
+        `max_tt_rank = r`
+      and
+        `max_tt_rank = [1] + [r] * (d-1) + [1]`
+    epsilon: a floating point number or None
```

Comment:
Epsilon is not supported at the moment. Raise NotImplementedError if it's not None.